airbyte-integrations/connectors/source-slack/source_slack/source.py

```diff
@@ -363,7 +364,8 @@ def streams(self, config: Mapping[str, Any]) -> List[Stream]:
         # this field is not exposed to spec, used only for testing purposes
         end_date = config.get("end_date")
         end_date = end_date and pendulum.parse(end_date)
-        threads_lookback_window = pendulum.Duration(days=config["lookback_window"])
+        threads_lookback_window = pendulum.Duration(days=config["threads_lookback_window"])
```

Comment:
We need to avoid braking changes and add migration config script. Please check google ads as a reference.

Reply:
May be migrate `lookback_window` to `threads_lookback_window` and set `messages_lookback_window` to 0 as default value.